### PR TITLE
enhancement/fix-password-vulnerability

### PR DESCRIPTION
**Pull request by cassa10**
## Description
- [x] Uncomment spring boot security from gradle but disable config by now.
- [x] Change unsecure password sha256 hash with bcrypt of spring boot security.
- [x] Add tests

### CODE REVIEW

File: back/src/main/kotlin/com/sos/smartopenspace/services/PasswordEncoderService.kt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package com.sos.smartopenspace.services
+
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder
+import org.springframework.security.crypto.password.PasswordEncoder
+import org.springframework.stereotype.Service
+
+@Service
+class PasswordEncoderService(
+    @Autowired
```
Comment (ngarbezza) [inline back/src/main/kotlin/com/sos/smartopenspace/services/PasswordEncoderService.kt:10]: me pregunto si es necesaria la annotation, no hay autowire por defecto?

File: back/src/test/kotlin/com/sos/smartopenspace/controllers/TalkControllerTest.kt [hunk 2]
```diff
@@ -267,7 +267,7 @@ class TalkControllerTest {
   private fun freeSlots(openSpace: OpenSpace) = openSpace.freeSlots().first().second
 
   private fun aReviewCreationBody(grade: Int, comment: String): String {
-    return return """
```
Comment (ngarbezza) [inline back/src/test/kotlin/com/sos/smartopenspace/controllers/TalkControllerTest.kt:270]: lol

**Review by ngarbezza — APPROVED** [review]
dejo un comentario no bloqueante!

File: back/src/main/kotlin/com/sos/smartopenspace/services/PasswordEncoderService.kt (hunk 1 above)
Comment (cassa10) [reply back/src/main/kotlin/com/sos/smartopenspace/services/PasswordEncoderService.kt:10]: creo que si, ahi lo arreglo

Comment (cassa10) [reply back/src/main/kotlin/com/sos/smartopenspace/services/PasswordEncoderService.kt]: Hecho

**Pull request merged**